contrib/services/systemd/dcrd.service

```diff
@@ -1,13 +1,53 @@
 [Unit]
 Description=Decred Full Node
+Requires=local-fs.target network-online.target network.target
+After=local-fs.target network-online.target network.target
 
 [Service]
 Type=simple
+
+# modify to your username
 User=dcrd
 Group=dcrd
-WorkingDirectory=/var/dcrd
-ExecStart=/opt/decred/bin/dcrd --appdata=/var/dcrd
+
+#dcrd appdir, %h will be replaced to ~/$User.
+ReadWritePaths=%h/.dcrd/
+
+# dcrd datadir, modify if data directory on other mount device. e.g /data/dcrd-data/
+ReadWritePaths=%h/.dcrd/data
+
+# dcrd logdir
+ReadWritePaths=%h/.dcrd/logs
+
+# launch dcrd
+ExecStart=%h/decred/dcrd
```

Comment:
Again, this (and thus the others) were intentionally not the user's home directory.  System-wide services are expected in certain locations and _not_ the user's home directory.